scripts/files/files_helper.py

```diff
@@ -39,6 +41,23 @@ def is_tiff(path: str) -> bool:
     return path.lower().endswith((".tiff", ".tif"))
 
 
+def is_GTiff(path: str) -> bool:
+    """Verifies if a file is a GTiff
+
+    Args:
+        path: a path to a file
+
+        Returns:
+        True if the file is a GTiff
+    """
+    gdal_data = gdal_info(path)
+    if gdal_data["cornerCoordinates"]["upperLeft"] == [0, 0]:
```

Comment:
there exists quite a few testing tiffs that have 0,0 as their origin, I have made a bunch as it makes math easier to verify...

Reply:
This was one of the option we found to differenciate a georeferenced TIFF to a non-georeferenced TIFF (example: `CB07_GeoTifv1-02.tif` VS `CB07_TIFFv1-02.tif`). If you think this can have an issue with some test data, there could be another option still using `gdalinfo` which is the `PROJCRS` that is not supposed to exist with a non-georeferenced TIFF.